README.md

```diff
@@ -31,11 +31,17 @@ Build and run StrongSwan on Linux
 	#build StrongSwan
 	./swanbuild_p4.sh -t native -o $DEPS_INSTALL --enable_grpc
 
-	#Run StrongSwan
+	#configure StrongSwan
+	Generate files(p4info, pb.bin) used to set p4 pipeline and place it in /var/tmp/
+	ls /var/tmp/linux_networking.p4info.txt
+	ls /var/tmp/ipsec_fixed_func.pb.bin
+
 	Generate the key and cert using openssl commands.
 	update the gnmi server IP and cert,key files path in ipsec_offload.conf and copy it in  /usr/share/stratum
 	cd output_strongswan
 	Update the etc/ipsec.conf, update the system date and stop StrongSwan service (systemctl stop strongswan.service)
```

Comment:
Lets not say "etc/ipsec.conf". The ipsec.conf file is deprecated now. 
We can say "Update the strongSwan configuration file(eg: swanctl.conf)"

Reply:
Updated it